### PR TITLE
[WIP] Move katello to rh-mongodb34

### PR DESCRIPTION
**Pull request by ehelms**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.17
* [ ] 1.16
* [ ] 1.15

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: packages/katello/katello/katello.spec [hunk 1]
```diff
@@ -51,8 +51,7 @@ Requires: pulp-puppet-tools
 Requires: pulp-selinux
 Requires: pulp-server
 Requires: python-pulp-streamer
-Requires: mongodb >= 2.4
-Requires: mongodb-server >= 2.4
+Requires: rh-mongodb34
```
Comment (evgeni) [inline packages/katello/katello/katello.spec]: Shouldn't this be `rh-mongodb34-mongodb`, as `rh-mongodb34` is the meta package for the SCL itself?

Comment (ehelms) [reply packages/katello/katello/katello.spec]: I think it can be -- from my testing both worked.

File: packages/katello/katello/katello.spec [hunk 2]
```diff
@@ -71,6 +70,8 @@ Requires: /usr/bin/psql
 Requires: /usr/bin/pg_dump
 Requires: /usr/bin/pg_dumpall
 
+Obsoletes: mongodb
```
Comment (ekohl) [inline packages/katello/katello/katello.spec]: This feels like a strong statement to make. In the past we haven't obsoleted elasticsearch in this way.

Comment (ehelms) [reply packages/katello/katello/katello.spec]: True, but we eliminated elasticsearch all together. In our case, we are wanting to force an upgrade. Further, the impression I was getting is that (https://github.com/Katello/katello-selinux/pull/15) will cause a conflict with the mongodb package?

File: packages/katello/katello/katello.spec (hunk 1 above)
Comment (chris1984) [reply packages/katello/katello/katello.spec]: looking here its https://www.softwarecollections.org/en/scls/rhscl/rh-mongodb34/

rh-mongodb34

File: packages/katello/katello/katello.spec (hunk 2 above)
Comment (chris1984) [reply packages/katello/katello/katello.spec]: The last thing we want to have is both SCL Mongo and Mongo 2.6 on the system at once.

Comment (ekohl) [reply packages/katello/katello/katello.spec]: If we add a Conflicts: in katello-selinux, would that remove the package as well or do we need to obsolete it somewhere?

Comment (ehelms) [reply packages/katello/katello/katello.spec]: Conflicts will prevent installation and requires manual user intervention.

**Comment by evgeni** [comment]
[test rpm]

**Comment by evgeni** [comment]
[test rpm]

**Comment by evgeni** [comment]
[test rpm]

**Comment by evgeni** [comment]
[test rpm]

**Comment by ekohl** [comment]
Should this be closed in favor of https://github.com/theforeman/foreman-packaging/pull/2343?

**Pull request closed without merging**